main.tf

```diff
@@ -0,0 +1,7 @@
+resource "serverless_deployment" "example" {
+  config_path         = abspath("serverless.yml")
+  # Relative to config path
+  package_dir         = ".terraform-serverless"
+  stage               = "sandbox"
+  serverless_bin_path = abspath("node_modules/.bin/serverless")
```

Comment:
Is this going to be the same on Windows and still executable? I usually see `{NAME}.(cmd|ps1)`

For example, in jetpack testing, we use this JavaScript:

```js
const IS_WIN = process.platform === "win32";
const SLS_CMD = `node_modules/.bin/serverless${IS_WIN ? ".cmd" : ""}`;
```